employee/members/views.py

```diff
@@ -0,0 +1,88 @@
+from django.shortcuts import render
+
+# Create your views here.
+from django.http import HttpResponse
+from django.template import loader  
+from django.views import View
+from .models import Employees
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.decorators import method_decorator
+import json
+#from .serializers import EmployeeSerializer
+
+
+def index(request):
+  template = loader.get_template('first.html')
+  return HttpResponse(template.render())
+  
+@method_decorator(csrf_exempt, name='dispatch')
+class EmployeeView(View):
+     def get(self, request, employee_id=None):
+          employee_model_list = []
+          try:
+             if employee_id:
+                employee_model_list = Employees.objects.filter(employee_id=employee_id)
+        
+          except Employees.DoesNotExist:
+             return JsonResponse({'status': 'failed', "employees": None}, status=400)
+          students = []
+          for student in employee_model_list:
+               data = {
+                   "first_name" : student.first_name,
+                   "last_name": student.last_name,
+                   "employee_salary": student.employee_salary,
+                   "employee_id": student.employee_id,
+                   "mobile": student.mobile
+                }
+               students.append(data)
+          if len(students)==0:
+              return JsonResponse({'status': 'failed', "employees": None}, status=400)
+          else:
+              return JsonResponse({'status': 'success', "employees": students}, status=200)
+
+     def post(self, request):
+          if not request.POST.get('first_name') or not request.POST.get('last_name') or not request.POST.get('employee_salary') or  not request.POST.get('employee_id') or not request.POST.get('mobile'):
+              return JsonResponse({'status': 'failed', "message" : "all fields required"}, status=500)
+
+          Employees.objects.create(
+              first_name= request.POST.get('first_name'),
+              last_name= request.POST.get('last_name'),
+              employee_salary= request.POST.get('employee_salary'),
+              employee_id= request.POST.get('employee_id'),
+              mobile= request.POST.get('mobile'))
+          return JsonResponse({'status': 'sucess'}, status=200)
+          
+     
+     def put(self, request):
+          
+          try:
+              data = json.loads(request.body)
+              employee_id=data['employee_id']
```

Comment:
@AnkitaHegde What if there is no employee id sent in the API request?

Reply:
@anushds , Thank you for noting it. I forgot to include that case. Now i have updated it.

Reply:
@AnkitaHegde You've done well with the commit message(among  all the PRs that I've reviewed). But it can be further improved.
You can look up in Google, how to write a good commit message.
So coming to the changes that you've done with respect to the comments that I had added, they look good. But you've addressed only 1 comment.
You should be addressing all the comments with the appropriate changes that you've done.

Reply:
@anushds ,Thank you . I will keep that in mind.